### PR TITLE
Clean up cookbooks to conform w JSON_INFO.md#books

### PR DESCRIPTION
**Pull request by Jerimee**
Clean up of cookbooks to conform with JSON_INFO.md

#### Summary
```SUMMARY: None```

#### Purpose of change

```
Clean up of cookbooks to conform with JSON_INFO.md

The json_info in the docs has a color key for books. The cooking.json (cookbooks) file didn't conform to this key.
```

#### Describe the solution

```
adv_chemistry is Large textbook so dark_blue
brewing_cookbook is Small textbook so dark_green
cookbook is Small textbook so dark_green
cookbook_human is Large textbook so dark_blue
cookbook_italian is Large textbook so dark_blue
cookbook_sushi is Small textbook so dark_green
family_cookbook is Printouts (including spiral-bound and similar) so light_green
modern_tanner is Small textbook so dark_green (was brown?)
reference_cooking is Large textbook so dark_blue
scots_cookbook is Large textbook so dark_blue
textbook_chemistry is Large textbook so dark_blue
```

#### Additional context

```I also made minor improvements to two cookbook description entries```

### CODE REVIEW

**Comment by Jerimee** [comment]
What in the world? Something I did made these checks all pissed off but I don't understand the detail logs...

**Comment by AMurkin** [comment]
These colors are not available. See [doc/COLOR.md](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/COLOR.md).

**Comment by Jerimee** [comment]
I saw this, but I must have misunderstood it.

"deprecated prefix lt instead of light_"

So is the correct way to specify a lighter shade of blue *ltblue* or
*light_blue* ?

Or am I missing something else entirely? I thought "deprecated" meant
disfavored or obsolete.

Is the doc I'm looking at itself out of date? I feel like it explicitly
prescribes using the "light_blue" value.

Thanks in advance for the help!

**Comment by Jerimee** [comment]
Or perhaps something like ltblue_black or light_blue_black? So as to make
the bg color obvious?

Thanks again for setting me straight

**Comment by AMurkin** [comment]
That's literally 16 [possible colors](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/COLOR.md#possible-colors).
The only dark color available is dark_gray.
You can use *Prefix_Foreground_Background* format.

**Comment by Jerimee** [comment]
Ok cool, and presumably this doc should be updated to reflect that?:

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_INFO.md#color-key

**Comment by Jerimee** [comment]
I'm going to leave this here uncorrected until I can get the doc itself updated - just out of an abundance of caution - my apologies in advance for leaving it open

**Comment by Jerimee** [comment]
Hooray, we updated the doc! Now to see if I can get this crossed off...

File: data/json/items/book/cooking.json [hunk 1]
```diff
@@ -3,13 +3,13 @@
     "id": "adv_chemistry",
     "type": "BOOK",
     "name": "Advanced Physical Chemistry",
-    "name_plural": "Advanced Physical Chemistry",
+    "name_plural": "Advanced Chemistry Textbooks",
```
Comment (cainiaowu) [inline data/json/items/book/cooking.json]:
```suggestion
    "name_plural": "copies of Advanced Physical Chemistry",
```

**Comment by I-am-Erk** [comment]
There are a *lot* more changes here than just colours. Can you go into more detail? Several seem to be fairly significant balance changes like adjusting the max level you can get from a text.

File: data/json/items/book/cooking.json [hunk 2]
```diff
@@ -171,34 +173,34 @@
     "skill": "cooking",
     "max_level": 1,
     "intelligence": 5,
-    "time": "24 m",
+    "time": "23 m",
```
Comment (I-am-Erk) [inline data/json/items/book/cooking.json]: Why the change in reading time?

File: data/json/items/book/cooking.json [hunk 3]
```diff
@@ -171,34 +173,34 @@
...
+    "time": "28 m"
   },
   {
     "id": "recipe_alpha",
     "type": "BOOK",
     "name": "PE050 \"Alpha\": Preliminary Report",
-    "name_plural": "PE050 \"Alpha\": Preliminary Report",
-    "description": "This sheaf of papers-dated two weeks before all this started-describes some new chemical formula, and its effects on human subjects.  It's stamped \"APPROVED\"…",
+    "name_plural": "PE050 \"Alpha\": Preliminary Reports",
+    "description": "This sheaf of papers is dated two weeks before all this started.  It describes some new chemical formula, and its effects on human subjects.  It's stamped \"APPROVED\"…",
```
Comment (I-am-Erk) [inline data/json/items/book/cooking.json]: There was nothing wrong with the original use of a dash, I'm not sure why the change.

File: data/json/items/book/cooking.json [hunk 4]
```diff
@@ -357,7 +360,7 @@
     "id": "recipe_raptor",
     "type": "BOOK",
     "name": "PE070 \"Raptor\": Proposal",
-    "name_plural": "PE070 \"Raptor\": Proposal",
+    "name_plural": "PE070 \"Raptor\": Papers",
```
Comment (I-am-Erk) [inline data/json/items/book/cooking.json]: This makes it appear that the name of the proposal changes if you're carrying two of it

File: data/json/items/book/cooking.json [hunk 5]
```diff
@@ -397,18 +400,18 @@
     "id": "reference_cooking",
     "type": "BOOK",
     "name": "CRC-Merck handbook, 4th edition",
-    "name_plural": "copies of the CRC-Merck handbook, 4th edition",
+    "name_plural": "4th edition copies of the CRC-Merck handbook",
```
Comment (I-am-Erk) [inline data/json/items/book/cooking.json]: This is a very confusing change.

File: data/json/items/book/cooking.json [hunk 6]
```diff
@@ -436,20 +439,21 @@
...
-    "bashing": 5,
+    "bashing": 4,
     "material": [ "paper" ],
     "symbol": "?",
     "color": "blue",
     "skill": "cooking",
     "required_level": 3,
     "max_level": 6,
-    "intelligence": 12,
+    "intelligence": 11,
```
Comment (I-am-Erk) [inline data/json/items/book/cooking.json]: Why was this changed?

File: data/json/items/book/cooking.json [hunk 7]
```diff
@@ -436,20 +439,21 @@
   {
     "id": "textbook_chemistry",
     "type": "BOOK",
-    "name": "chemistry textbook",
-    "description": "A college textbook on chemistry.",
+    "name": "Chemistry Textbook",
+    "name_plural": "Chemistry Textbooks",
+    "description": "This college textbook of practical organic chemistry explains the basic applications of chemistry to everyday living.",
```
Comment (I-am-Erk) [inline data/json/items/book/cooking.json]: This doesn't sound like any college organic chem text I've ever seen. I'd say "This college organic chemistry textbook drily explains in excruciating detail dozens of chemical reaction mechanisms both in theory and in practice."

Comment (anothersimulacrum) [reply data/json/items/book/cooking.json]:
```suggestion
    "description": "This college organic chemistry textbook dryly explains in excruciating detail dozens of chemical reaction mechanisms both in theory and in practice.",
```

File: data/json/items/book/cooking.json (hunk 4 above)
Comment (anothersimulacrum) [reply data/json/items/book/cooking.json]:
```suggestion
    "name_plural": "PE070 \"Raptor\": Proposals",
```

File: data/json/items/book/cooking.json (hunk 2 above)
Comment (Jerimee) [reply data/json/items/book/cooking.json]: arbitrary! this change of a minute less in time to read is an unnecessary distraction, it ought not to have been included!!

File: data/json/items/book/cooking.json (hunk 6 above)
Comment (Jerimee) [reply data/json/items/book/cooking.json]: for no good reason

File: data/json/items/book/cooking.json (hunk 7 above)
Comment (Jerimee) [reply data/json/items/book/cooking.json]: This might not be relevant, but this specific edit was inspired after reviewing a half dozen  chemistry textbooks published between 1945 and 1965. It is trivial and can be eliminated without second thoughts. My intent was to provide amusing copy.

I believe the original was _"A college textbook on chemistry."_

**Comment by Jerimee** [comment]
It seems I muddied the waters here. Should I edit this PR so as to include simply the color specifications? I apologize for additional modifications beyond those specified. 

I did make a number of small tweaks to various values - in my mind these were to calibrate imbalances with the original content.  I have neither inclination nor justification to insist upon these changes. 

Please advise. Thanks in advance! :)

File: data/json/items/book/cooking.json (hunk 1 above)
Comment (Jerimee) [reply data/json/items/book/cooking.json]: uh, I saw something in the discord about less characters being better... the alternative you've provided is perfectly nice by me. Thanks for the help :)

**Comment by I-am-Erk** [comment]
I would generally advise against a bunch of changes for change's sake, and in general you shouldn't be making any changes that aren't described in the PR itself. Reviewers read the PR description to determine if there's stuff they should look at, and nothing about your PR description makes it sound like you're adjusting book difficulty levels and skill gains, let alone a bunch of random shifts in reading times and melee attack damages and things that are completely arbitrary.

**Comment by DaviBones** [comment]
Yeah this is a mess. I think you should pare it down to just the color changes, and *maybe* the description/plural changes. You can make the numerical changes, if you're prepared to justify them, in a separate PR.

**Comment by Jerimee** [comment]
Sounds good
